doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1101,6 +1101,7 @@ Reshaping
 - Bug in :func:`crosstab` when inputs are two Series and have tuple names, the output will keep dummy MultiIndex as columns. (:issue:`18321`)
 - :meth:`DataFrame.pivot` can now take lists for ``index`` and ``columns`` arguments (:issue:`21425`)
 - Bug in :func:`concat` where the resulting indices are not copied when ``copy=True`` (:issue:`29879`)
+- Bug in :meth:`SeriesGroupBy.aggregate` was resulting in aggregations being overwritten when they shared the same name (:issue:`30880`)
```

Comment:
FYI: the link to this method won't render, since `SeriesGroupBy` isn't in the pands namespace.

Reply:
Sorry about that - will make sure the build the whatsnew file in the future to check